src/create-cluster.ts

```diff
@@ -42,19 +42,36 @@ export async function createCluster(
     runtime = params['minikube.cluster.creation.runtime'];
   }
 
+  let baseImage = '';
+  if (params['minikube.cluster.creation.base-image']) {
+    baseImage = params['minikube.cluster.creation.base-image'];
+  }
+
+  let mountString = '';
+  if (params['minikube.cluster.creation.mount-string']) {
+    mountString = params['minikube.cluster.creation.mount-string'];
+  }
```

Comment:
```suggestion
  conts mountString = params['minikube.cluster.creation.mount-string'] ?? '';
```

Reply:
so I don't follow previous pattern for other params.

should I replace all previous lines before ?

Reply:
anyway I think we don't need the `?? ''`;

Reply:
> anyway I think we don't need the `?? ''`;

Yes, right, as the result will be `undefined`, and will be caught later with  `if (mountString)`.

My main concern here is to avoid to repeat the same key twice (typo is in the corner ;) )

Reply:
> so I don't follow previous pattern for other params.
> 
> should I replace all previous lines before ?

The pattern is a little different for previous flags, as there is a 'real' default value

Reply:
fixed